api/api/settings/prod.py

```diff
@@ -29,7 +29,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = False
 
-ALLOWED_HOSTS = list(os.environ.get("DJANGO_ALLOWED_HOSTS", "").split(" "))
+CORS_ALLOW_HEADERS = "*"
```

Comment:
allow all headers could potentially cause security issues in prod. Please specify exactly the headers you need.